toolkit/tools/imagegen/configuration/user.go

```diff
@@ -87,6 +91,16 @@ func (p *User) UIDIsValid() (err error) {
 	return
 }
 
+// This function does not account for the Password key being intentionally
+// unspecified versus set to blank
+// PasswordIsValid returns an error if the User password is empty
+func (p *User) PasswordIsValid() (err error) {
+	if p.Password == "" && p.Name != "root" {
+		return fmt.Errorf("invalid value for Password (%s)", p.Password)
```

Comment:
Recommend rewording error comment here. Password will always be blank.

Reply:
Changed to say empty